Components v2 (#7487)
hammerandchisel/discord-api-docs: docs/resources/message.md

| LOADING                                | `1 << 7`  | this message is an Interaction Response and the bot is "thinking"                                                                                          |
| FAILED_TO_MENTION_SOME_ROLES_IN_THREAD | `1 << 8`  | this message failed to mention some roles and add their members to the thread                                                                              |
| SUPPRESS_NOTIFICATIONS                 | `1 << 12` | this message will not trigger push and desktop notifications                                                                                               |
| IS_VOICE_MESSAGE                       | `1 << 13` | this message is a voice message                                                                                                                            |
| HAS_SNAPSHOT                           | `1 << 14` | this message has a snapshot (via Message Forwarding)                                                                                                       |

> | IS_COMPONENTS_V2                       | `1 << 15` | allows you to create fully [component](/docs/components/reference#component-object) driven messages while removing `content` and `embeds` from the message |

Reviewer: The description does not fit the same tone/delivery as the other descriptions.

> | IS_COMPONENTS_V2                       | `1 << 15` | allows you to create fully [component](/docs/components/reference#component-object) driven messages while removing `content` and `embeds` from the message |

Reviewer 2: ```suggestion
| IS_COMPONENTS_V2                       | `1 << 15` | this message is using [component](/docs/components/reference#component-object) driven messages, effectively removing `content` and `embeds` |
```

or something like that
